apps/console/src/features/server-configurations/forms/login-attempt-security-form.tsx

```diff
@@ -532,6 +557,25 @@ export const LoginAttemptSecurityConfigurationFrom: FunctionComponent<
                 {
                     sampleInfoSection()
                 }
+                <Field.Checkbox
+                    ariaLabel="notifyUserOnAccountLockIncrement"
+                    name="notifyUserOnAccountLockIncrement"
+                    label={ t("extensions:manage.serverConfigurations.accountSecurity." +
+                    "loginAttemptSecurity.form.fields.notifyUserOnAccountLockIncrement.label") }
+                    listen={ (value: boolean) => setNotifyUserOnAccountLockIncrement(value) }
+                    checked={ notifyUserOnAccountLockIncrement }
+                    required={ false }
+                    readOnly={ readOnly }
+                    disabled={ !isConnectorEnabled }
+                    width={ 10 }
+                    data-testid={ `${testId}-notify-user-on-account-lock-time-increase` }
+                />
+                <Hint className={ "mb-5" }>
```

Comment:
refrain from using helper classes for margin's. For hint, you can use the `hint` prop in the `Field.Checkbox`